opensearch-operator/opensearch-gateway/services/os_client.go

```diff
@@ -122,29 +123,6 @@ func (client *OsClusterClient) CatIndices() ([]responses.CatIndicesResponse, err
 	return response, err
 }
 
-func (client *OsClusterClient) CatSystemIndices() ([]responses.CatIndicesResponse, error) {
```

Comment:
Yeah, the way the cat operations work is that if one of the indices passed in doesn't exist it will get a 404 response.  So we need to check each system index string on its own to make sure it exists, and then we pass a list of system indices that we know exist to CatShards